problems/objects.js

```diff
@@ -177,7 +243,18 @@ const countAandE = () => {};
  * @returns {Object} Counts of all characters: {a: 2, g: 1, o: 2, d:1, " ": 2, s: 1, n:1, k:1, e: 1}
  */
 
-const countOccurance = () => {};
+const countOccurance = (str) => {
+  let occurence = {};
+  for (let i = 0; i < str.length; i++) {
+    const el = str[i].toLowerCase();
+    if (occurence[el] !== undefined) {
```

Comment:
occurence[el] also works on it's own, because if it evaluates to undefined it will be falsey already.